src/script/externalRoute.ts

```diff
@@ -20,79 +20,41 @@
-  },
-  WEBSITE: env.URL?.WEBSITE_BASE,
-};
-
-export const URL_PATH = {
-  CREATE_TEAM: '/create-team/',
-  DECRYPT_ERROR_1: '/articles/207948115',
-  DECRYPT_ERROR_2: '/privacy/error-2/',
-  MANAGE_SERVICES: '/services/',
-  MANAGE_TEAM: '/login/',
-  PASSWORD_RESET: '/forgot/',
-  PRIVACY_HOW: '/privacy/how/',
-  PRIVACY_UNVERIFIED_USERS: '/articles/202857164',
-  PRIVACY_WHY: '/articles/207859815',
-  SUPPORT_USERNAME: '/support/username/',
-} as const;
+const URL = Config.getConfig().URL;
 
 const getTeamSettingsUrl = (path: string = '', utmSource?: string): string | undefined => {
   const query = utmSource ? `?utm_source=${utmSource}&utm_term=desktop` : '';
-  const teamSettingsUrl = `${URL.TEAM_SETTINGS}${path}${query}`;
-  return URL.TEAM_SETTINGS ? teamSettingsUrl : undefined;
+  const teamSettingsUrl = `${URL.TEAMS_BASE}${path}${query}`;
+  return URL.TEAMS_BASE ? teamSettingsUrl : undefined;
 };
 
-export const getWebsiteUrl = (path: string = '', pkCampaign?: string): string | undefined => {
-  if (URL.WEBSITE) {
+const getWebsiteUrl = (path: string = '', pkCampaign?: string): string | undefined => {
+  if (URL.WEBSITE_BASE) {
     const query = pkCampaign ? `?pk_campaign=${pkCampaign}&pk_kwd=desktop` : '';
-    const websiteUrl = `${URL.WEBSITE}${path}${query}`;
-    return addLocaleToUrl(URL.WEBSITE ? websiteUrl : undefined);
+    const websiteUrl = `${URL.WEBSITE_BASE}${path}${query}`;
+    return addLocaleToUrl(URL.WEBSITE_BASE ? websiteUrl : undefined);
   }
   return undefined;
 };
 
-const getHelpCenterUrl = (path: (typeof URL_PATH)[keyof typeof URL_PATH]) => {
-  if (URL.SUPPORT) {
-    const helpcenterUrl = `${URL.SUPPORT}${path}`;
-    return addLocaleToHelpCenterUrl(URL.SUPPORT ? helpcenterUrl : undefined);
-  }
-  return undefined;
+const getAccountPagesUrl = (path: string = ''): string | undefined => {
+  const accountPagesUrl = `${URL.ACCOUNT_BASE}${path}`;
+  return URL.ACCOUNT_BASE ? accountPagesUrl : undefined;
 };
 
-export const getAccountPagesUrl = (path: string = ''): string | undefined => {
-  const accountPagesUrl = `${URL.ACCOUNT}${path}`;
-  return URL.ACCOUNT ? accountPagesUrl : undefined;
-};
+const getPrivacyPolicyUrl = (): string | undefined => addLocaleToUrl(URL.PRIVACY_POLICY || undefined);
+const getTermsOfUsePersonalUrl = (): string | undefined => addLocaleToUrl(URL.TERMS_OF_USE_PERSONAL || undefined);
+const getTermsOfUseTeamUrl = (): string | undefined => addLocaleToUrl(URL.TERMS_OF_USE_TEAMS || undefined);
 
-export const getPrivacyPolicyUrl = (): string => addLocaleToUrl(URL.PRIVACY_POLICY || undefined);
-export const getTermsOfUsePersonalUrl = (): string => addLocaleToUrl(URL.TERMS_OF_USE_PERSONAL || undefined);
-export const getTermsOfUseTeamUrl = (): string => addLocaleToUrl(URL.TERMS_OF_USE_TEAMS || undefined);
+export const getManageServicesUrl = (utmSource?: string): string | undefined =>
+  getTeamSettingsUrl(URL.URL_PATH.MANAGE_SERVICES, utmSource);
```

Comment:
hard to find doc on those UTM sources
we use 2 variations of those in the webapp:
- `client_landing` when navigating from the main portion of the app
- `client_settings` when navigating from the settings

Do we want to keep that granularity @atomrc ?
Should I had those to the `externalUrl` object and stop exporting those methods?

Reply:
I'll keep the export of the method and add some JSDoc to make the utmSource thing more transparent, how does that sound?

Reply:
That sounds good to me :)